fixtures/transxchange2ntfs/input/config.json

```diff
@@ -0,0 +1,18 @@
+{
+	"contributor": {
+		"contributor_id": "UK",
```

Comment:
is it a tab or 8 spaces?

Reply:
It's tabulations, why?

Reply:
we use 4 spaces for all

Reply:
This is a copy from https://github.com/CanalTP/transit_model/blob/master/fixtures/kv12ntfs/config.json which also contains tabulations. Should I change it everywhere?

Reply:
yes please.

Reply:
Done in f0214eb25ee98a81b04edcc11762b464668340cc.